### PR TITLE
Updated Kubernetes dependency to v1.15 and fixed all dep errors

### PR DESCRIPTION
**Pull request by davidz627**
The `Gopkg.toml` file is now sorted and commented and all makes sense.

/assign @msau42

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -24,65 +24,67 @@
...
 
 [[constraint]]
-  name = "google.golang.org/grpc"
-  version = "1.10.0"
+  version = "v1.15.0"
+  name = "k8s.io/kubernetes"
 
 [[constraint]]
-  branch = "release-1.11"
+  version = "kubernetes-1.15.0-alpha.3"
```
Comment (msau42) [inline Gopkg.toml]: Is there an official 1.15 tag we can use now

Comment (davidz627) [reply Gopkg.toml]: yeah looks like they were just published

**Comment by davidz627** [comment]
@msau42 I believe I've resolved all remaining dep issues (mainly around the `klog` import) and all tests should be passing soon :)

**Pull request merged**